x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -391,5 +393,34 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         expect(await latestFindingsTable.getFindingsCount('passed')).to.eql(passedFindingsCount);
       });
     });
+
+    describe('Access with custom roles', async () => {
+      before(async () => {
```

Comment:
I am not very familiar with the FTRs right now but is there a higher-level test suite callback function we can use so we don't need to repeat creating and cleaning roles and users?

Reply:
good point, it relates to what Kfir mentioned [here](https://github.com/elastic/kibana/pull/184622#discussion_r1624617157). 

I agree and now [it creates](https://github.com/elastic/kibana/pull/184622/files#diff-7863c190bc746eb97a0db2090dd0c48915e91731811688621e5f48f68115f098R14) users and roles only once.